<ui version="4.0" >
 <class>TupletDialog</class>
 <widget class="QDialog" name="TupletDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MuseScore: create tuplet</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="mscore.qrc" >:/data/bg1.jpg</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>9</number>
     </property>
     <property name="topMargin" >
      <number>9</number>
     </property>
     <property name="rightMargin" >
      <number>9</number>
     </property>
     <property name="bottomMargin" >
      <number>9</number>
     </property>
     <item row="0" column="0" >
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Type</string>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" >
        <property name="leftMargin" >
         <number>9</number>
        </property>
        <property name="topMargin" >
         <number>9</number>
        </property>
        <property name="rightMargin" >
         <number>9</number>
        </property>
        <property name="bottomMargin" >
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>actual notes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="actualNotes" >
          <property name="minimum" >
           <number>2</number>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
          <property name="value" >
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>normal notes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="normalNotes" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
          <property name="value" >
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Format</string>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>9</number>
        </property>
        <property name="leftMargin" >
         <number>6</number>
        </property>
        <property name="topMargin" >
         <number>6</number>
        </property>
        <property name="rightMargin" >
         <number>6</number>
        </property>
        <property name="bottomMargin" >
         <number>6</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_3" >
          <property name="title" >
           <string>Number</string>
          </property>
          <property name="flat" >
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" >
           <property name="leftMargin" >
            <number>6</number>
           </property>
           <property name="topMargin" >
            <number>6</number>
           </property>
           <property name="rightMargin" >
            <number>6</number>
           </property>
           <property name="bottomMargin" >
            <number>6</number>
           </property>
           <item>
            <widget class="QRadioButton" name="number" >
             <property name="text" >
              <string>number</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="relation" >
             <property name="text" >
              <string>relation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="noNumber" >
             <property name="text" >
              <string>nothing</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4" >
          <property name="title" >
           <string>Bracket</string>
          </property>
          <property name="flat" >
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" >
           <property name="leftMargin" >
            <number>6</number>
           </property>
           <property name="topMargin" >
            <number>6</number>
           </property>
           <property name="rightMargin" >
            <number>6</number>
           </property>
           <property name="bottomMargin" >
            <number>6</number>
           </property>
           <item>
            <widget class="QRadioButton" name="autoBracket" >
             <property name="text" >
              <string>auto bracket</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="bracket" >
             <property name="text" >
              <string>bracket</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="noBracket" >
             <property name="text" >
              <string>nothing</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mscore.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TupletDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TupletDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
